Library/OcPciIoLib/OcPciIoLib.c

```diff
@@ -0,0 +1,73 @@
+/** @file
+   PciIo overrides
+
+   Copyright (c) 2023, xCuri0. All rights reserved.<BR>
+   SPDX-License-Identifier: BSD-2-Clause-Patent
+
+ **/
+
+#include "OcPciIoU.h"
+#include <Library/OcPciIoLib.h>
+
+EFI_CPU_IO2_PROTOCOL *
+OcPciIoInstallProtocol (
+  IN BOOLEAN  Reinstall
+  )
+{
+  EFI_TPL                          Tpl;
+  UINTN                            Index;
+  EFI_STATUS                       Status;
+  EFI_CPU_IO2_PROTOCOL             *CpuIo;
+  UINTN                            HandleCount;
+  EFI_HANDLE                       *HandleBuffer;
+  EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL  *PciRootBridgeIo;
+
+  CpuIo = InitializeCpuIo2 ();
+  if (!CpuIo) {
```

Comment:
I think the code style for EDK is CpuIo == NULL, i.e. explicitly compare against actual value for all argument types but BOOLEAN.